### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by TwoMoreThings**
Added note about ensuring you inherit from SurfaceController and added the name to use for the partial view
Also changed wording around need to reload macros as this will hopefully be fixed in future

### CODE REVIEW

File: Getting-Started/Code/Creating-Forms/index.md [hunk 1]
```diff
@@ -37,6 +37,8 @@ Build your solution after adding the model.
 ### Creating the view
 Next, we add the view for the form to the `/View/Partials` folder. Because we've added the model and built the solution we can add it as a strongly typed view.
 
+Name your view "ContactForm"
```
Comment (sofietoft) [inline Getting-Started/Code/Creating-Forms/index.md]:
```suggestion
Name your view "ContactForm".
```

File: Getting-Started/Code/Creating-Forms/index.md [hunk 2]
```diff
@@ -86,6 +88,8 @@ namespace MyFirstForm.Controllers
 }
 ```
 
+You should note that the controller inherits "SurfaceController" and not from "Controller" as initially added to the file by Visual Studio
```
Comment (sofietoft) [inline Getting-Started/Code/Creating-Forms/index.md]:
```suggestion
You should note that the controller inherits "SurfaceController" and not from "Controller" as initially added to the file by Visual Studio.
```

File: Getting-Started/Code/Creating-Forms/index.md [hunk 3]
```diff
@@ -119,7 +123,9 @@ In the partial view, we're going to render our contact form using the view model
 
 
 #### Adding the macro
-The last thing to do before we can add the form to a page is to **allow the Macro in a rich text editor**. Right click __Macros__ and select __Reload__ then expand the __Macros__ node and select the __Contact Form__ Macro. Check the boxes under __Editor Settings__.
+The last thing to do before we can add the form to a page is to **allow the Macro in a rich text editor**. Expand the __Macros__ node and select the __Contact Form__ Macro. Check the boxes under __Editor Settings__.
+
+Tip: If you don't see your new macro listed, right click __Macros__ and select __Reload__
```
Comment (sofietoft) [inline Getting-Started/Code/Creating-Forms/index.md]: Excellent idea putting this into a tip here! 👏 

Have you seen the formatting options we have for tips like these?
You can see them here: https://our.umbraco.com/documentation/Contribute/Markdown-Conventions/#adding-notes-warnings-tips 
Makes the tip stands out a bit more! What do you think?

**Comment by TwoMoreThings** [comment]
Thanks Sophie,
that's great
regards
t

**Comment by sofietoft** [comment]
Awesome! 😄 
Would you mind committing the suggestions?
I don't have permissions to do that on your patch 🙈

**Comment by sofietoft** [comment]
Thanks again @TwoMoreThings 😄 🎉

**Pull request merged**